tests/src/test/scala/system/rest/SwaggerTests.scala

```diff
@@ -46,6 +46,20 @@ class SwaggerTests extends FlatSpec with Matchers with RestUtil {
     response.body().asString().contains("\"swagger\":") should be(true)
   }
 
+  it should "respond to /api-docs including ActionMeta/ActionExecMeta/RuleMeta/TriggerMeta" in {
+    val response = RestAssured.given().config(sslconfig).get(getServiceURL() + "/api/v1/api-docs")
+
+    response.statusCode() should be(200)
+    response.body().asString().contains("\"#/definitions/ActionMeta\"") should be(true)
```

Comment:
Since the controller just returns the content of the swagger JSON file, I am not quite sure it is meaningful.
We need to better way to make the swagger JSON file conforms to the actual APIs controllers provide.

But have no idea yet..